### PR TITLE
Update UG project config with create project tutorial

### PR DESCRIPTION
**Pull request by FiniteStateGit**
@netlify /docs/user-guide/project-config/project-manager/

## Change summary

- Update https://www.o3de.org/docs/user-guide/project-config/project-manager/ with tutorial on creating a new project.
- Update tutorial in Welcome Guide to reflect UI changes.
- Update media.

### Submission Checklist:

* [X] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [X] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [X] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [X] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Review by willihay — APPROVED** [review]
Looks good. Thanks for all the screenshot updates too.

File: content/docs/user-guide/project-config/project-manager.md [hunk 1]
```diff
@@ -155,3 +155,96 @@ The Edit Project Settings screen contains the following project settings:
...
+
+    When the build completes, you can find the project binaries in your project directory under `build/<platform>/bin/profile`.
+    
+    {{< note >}}
+**(Development/22.10 release branch)** On Windows, if you have multiple versions of Visual Studio installed, Project Manager will build with the highest version detected. To specify the version of Visual Studio, use the CMake environment variable `CMAKE_GENERATOR_PLATFORM` with a [value from the CMake generator list](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html#visual-studio-generators).
+    {{< /note >}}
+
+    {{< note >}}
+On Windows, if you have multiple versions of Visual Studio installed, Project Manager will build with the highest version detected. To specify the version of Visual Studio, use the CMake environment variable `CMAKE_GENERATOR_PLATFORM` with a [value from the CMake generator list](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html#visual-studio-generators).
+    {{< /note >}}
```
Comment (micronAMZN) [inline content/docs/user-guide/project-config/project-manager.md:246]: Is it possible to combine these two notes?

**Review by micronAMZN — APPROVED** [review]
Just one comment about combining two nearly identical notes.

File: content/docs/user-guide/project-config/project-manager.md [hunk 2]
```diff
@@ -155,3 +155,96 @@ The Edit Project Settings screen contains the following project settings:
...
+    {{< note >}}
+The first build may take some time to complete if the required third-party packages need to be downloaded.
+    {{< /note >}}
+
+    When the build completes, you can find the project binaries in your project directory under `build/<platform>/bin/profile`.
+    
+    {{< note >}}
+**(Development/22.10 release branch)** On Windows, if you have multiple versions of Visual Studio installed, Project Manager will build with the highest version detected. To specify the version of Visual Studio, use the CMake environment variable `CMAKE_GENERATOR_PLATFORM` with a [value from the CMake generator list](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html#visual-studio-generators).
+    {{< /note >}}
+
```
Comment (FiniteStateGit) [inline content/docs/user-guide/project-config/project-manager.md:243]:
```suggestion
```

**Pull request merged**